.github/workflows/ci.yml

```diff
@@ -8,11 +8,12 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
 
     strategy:
       matrix:
-        node-version: [20.x]
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        node-version: [20.x, 22.x, 24.x]
```

Comment:
Running build and test jobs against multiple Node versions is a good idea. I don't think the OS matrix is terribly valuable though.